sale_commercial_partner/report/sale_report.py

```diff
@@ -11,12 +11,19 @@ class SaleReport(models.Model):
     commercial_partner_id = fields.Many2one(
         'res.partner', string='Commercial Customer', readonly=True)
 
-    def _select(self):
-        sql_str = super(SaleReport, self)._select()
-        sql_str += ', s.commercial_partner_id as commercial_partner_id'
-        return sql_str
+    # def _select(self):
```

Comment:
Have seen that Odoo 9.0 already puts `partner.commercial_id` in this view, so this part can indeed be removed.